azkaban-common/src/main/java/azkaban/utils/ExecuteAsUserUtils.java

```diff
@@ -0,0 +1,57 @@
+package azkaban.utils;
+
+import azkaban.jobExecutor.utils.JobExecutionException;
+import java.util.Arrays;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class is used as utility class for ExecuteAsUser script.
+ */
+public class ExecuteAsUserUtils {
+
+  private static final Logger logger = LoggerFactory.getLogger(ExecuteAsUserUtils.class);
+  private static final String ADD_GROUP_PATH = "/usr/sbin/groupadd";
```

Comment:
Perhaps this should be configurable as well, but I'm happy with that being in a future PR.

Reply:
Plus one. If we do it, better we do it for all the bin paths like CHOWN and CHMOD. Let's take it later.